var/spack/repos/builtin/packages/rmgdft/package.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Rmgdft(CMakePackage, CudaPackage, ROCmPackage):
+    """RMGDFT is a high performance real-space density functional code
+       designed for large scale electronic structure calculations."""
+
+    homepage = "http://www.rmgdft.org/"
+    git      = "https://github.com/RMGDFT/rmgdft.git"
+    maintainers = ['elbriggs']
+    tags = ['ecp', 'ecp-apps']
+    version('master')
+
+    variant('build_type', default='Release',
+            description='The build type to build',
+            values=('Debug', 'Release', 'RelWithDebInfo'))
+    variant('cuda', default=False,
+            description='Build with cuda support.')
+    variant('hip', default=False,
+            description='Build with hip/rocm support.')
```

Comment:
I think this serves the same purpose as the `rocm` variant inherited from `ROCmPackage`. Can you double check and eventually remove this one?

Reply:
After some internal discussion we've decided because of some instabilities and issues with gpu builds to only use cpu builds in spack for now. I will go ahead and close this PR and submit a new one with updates.